modules/js/test/test_calib3d.js

```diff
@@ -0,0 +1,109 @@
+// //////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Please replace to the shorter version of header:

https://github.com/cameron-martin/opencv/blob/41018e0cb7375ccf87bd0af0df3e6755812fea78/modules/js/test/test_features2d.js

Reply:
Fixed.